great_expectations/data_context/types/base.py

```diff
@@ -1629,6 +1629,7 @@ class DataContextConfigSchema(Schema):
         allow_none=True,
     )
     fluent_datasources = fields.Dict(
+        keys=fields.Str(),
```

Comment:
Why is this changing?

Reply:
@Kilo59 Style -- nothing special here.  Just indicating the keys are dictionaries.